passfort.gemspec

```diff
@@ -24,5 +24,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "gc_ruboconfig", "~> 2.3"
   spec.add_development_dependency "pry", "~> 0.10"
   spec.add_development_dependency "rspec", "~> 3.2"
+  spec.add_development_dependency "rspec_junit_formatter", "~> 0.3"
   spec.add_development_dependency "webmock", "~> 3.3"
 end
```

Comment:
A couple of notes:
- should we mention `Ruby client for...` in summary?
- should we add description as well?

Reply:
Probably! Let's do that in a separate PR though? It's unrelated to CI.